spmd/compiler/log_utils.py

```diff
@@ -17,3 +20,75 @@ def rank0_info(logger: logging.Logger, *args: Any, **kwargs: Any) -> None:
 def rank0_warning(logger: logging.Logger, *args: Any, **kwargs: Any) -> None:
```

Comment:
nit: delete the warning too?

Reply:
This is a followup task since I want to make sure this current logger logic works. Backup for devs in case it does not.